testing/trino-product-tests/src/main/java/io/trino/tests/hive/TestHiveStorageFormats.java

```diff
@@ -480,6 +480,41 @@ public void testSnappyCompressedParquetTableCreatedInHive()
         onHive().executeQuery("DROP TABLE " + tableName);
     }
 
+    @Test(groups = STORAGE_FORMATS)
+    public void testOrcStructsWithNonLowercaseFields()
+    {
+        String tableName = "orc_structs_with_non_lowercase";
+
+        ensureDummyExists();
+        onHive().executeQuery("DROP TABLE IF EXISTS " + tableName);
+
+        onHive().executeQuery(format(
+                "CREATE TABLE %s (" +
+                        "   c_bigint BIGINT," +
+                        "   c_struct struct<testCustId:string, requestDate:string>)" +
+                        "STORED AS ORC ",
+                tableName));
+
+        onHive().executeQuery(format(
+                "INSERT INTO %s"
+                // insert with SELECT because hive does not support array/map/struct functions in VALUES
+                + " SELECT"
+                + "   1,"
+                + "   named_struct('testCustId', '1234', 'requestDate', 'some day')"
+                // some hive versions don't allow INSERT from SELECT without FROM
+                + " FROM dummy",
+                tableName));
+
+        setProjectionPushdownEnabled(onTrino().getConnection(), true);
+        assertThat(query("SELECT c_struct.testCustId FROM " + tableName)).containsOnly(row("1234"));
+        assertThat(query("SELECT c_struct.testcustid FROM " + tableName)).containsOnly(row("1234"));
+        assertThat(query("SELECT c_struct.requestDate FROM " + tableName)).containsOnly(row("some day"));
+        setProjectionPushdownEnabled(onTrino().getConnection(), false);
```

Comment:
`onTrino.getConnection()` and `query(` use different connections, so seems like disabling the pushdown doesn't get applied really. Can you use `onTrino().query`? (Or something similar to `AbstractTestHiveViews#setSessionProperty`)